lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -405,6 +406,9 @@ def _configure_node(
     ) -> None:
         node_context = get_node_context(node)
 
+        if node_runbook.ignition:
```

Comment:
It would probably be goodness to ensure that `ignition` and `cloud_init` are not specified at the same time.